HRC_TXING_CHECK/HRC_Txing_Check.py

```diff
@@ -30,58 +50,304 @@
 WTxxxxxxxx   <--- The last digits of the parameter block identifies the SI mode
 XTZ0000005    <---- Start Science Command; would be XCZ if it's continuous clocking
 
-The structure is identical for all  SI modes. The only items that change from one SI mode
-to another are the second WSPOWxxxxx, the WTxxxxxxxx, and the Start Science command.
-
 This program assumes that no ACISPKT command will be inserted inside the block of SI mode
 commands by mission planning.  It would be illegal for any the 8 commands of the SI mode to be 
-moved with respect to each other. This can never happen because ACIS has created the the SI modes
-as a unit and the load builder loads the SI modes as a unit.
+moved with respect to each other. This shouldn't happen because ACIS has created the the SI modes
+as a unit and the load builder loads the SI modes as a unit. However the program checks the sequence
+and timing of the commands in case a hand edit of the load introduced an error.
+
+The structure is identical for all with bias SI modes. The only items that change from one SI mode
+to another are the second WSPOWxxxxx, the WTxxxxxxxx, and the Start Science command.
```

Comment:
added that possibility to the comment.

Reply:
As for the no-bias modes:

Yes TODAY everyone is set up to only use with-bias for this.

But things change and so can that. I wanted to be sure the program worked should someone use a no-bias version.  I don't want to have to go back and modify/retest everything because things change.  I verified in my testing that it does work. So I removed the no-bias dat files and no-bias entries in the map file. Should, somehow, we need to use no-bias in the future it's a simple addition to the .dat files and the mapping file.

Reply:
Yes I'll be able to  confirm consistency between what MP is  doing with the timing and what the code is checking.